applications/browser/package.json

```diff
@@ -0,0 +1,120 @@
+{
+  "private": true,
+  "name": "theia-blueprint-node",
```

Comment:
theia-blueprint-browser? Though a private package, we still see the name when e.g. running `yarn why pkg`